attributioncode/validator.go

```diff
@@ -73,6 +74,7 @@ type Code struct {
 	UA            string
 	ClientID      string
 	SessionID     string
+	DlSource      string
```

Comment:
What do you think about using a slightly longer attribute name?

```suggestion
	DownloadSource      string
```

Reply:
Up to you, as far as I'm concerned.